README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
+_marketplace builder credentials_ and the data_bag should _never_ be committed into
+source control.
+
+## Build Marketplace Images
+
+Test Kitchen has several suites for publishing different combinations of images.
+
+If you're targeting a specific release for a cloud and/or product set you'll
+probably want to use a targeted combination.
+
+Run kitchen list to view all the release targets:
+
+```shell
+kitchen list
+```
 
 Run the kitchen suite for the products that you wish to build, eg:
 
 ```shell
 kitchen converge aws-all-publish
 ```
 
+## Test Marketplace Images
+
+Currently there is no automated acceptance testing of marketplace images.
+In general what we've done is launch one of more of the newly created images
+and follow the setup [AWS](docs.chef.io/aws_marketplace.html) or
+[Azure](https://docs.chef.io/azure_portal.html#azure-marketplace) setup
+documentation to make sure that the user can login and start using the software.
+
+## Release Marketplace Images
+
+After the images have been built there are several manual steps that must be
+done to release the images in their respective marketplaces.
+
+### AWS Release
+
+After the images have been built you'll need to run them through the AWS
+Marketplace security scanner.
+
+1. Log into into the [AWS Marketplace](https://aws.amazon.com/marketplace/management/manage-products/?#/manage-amis.unshared) using the 'aws-marketplace' credentials in lastpass.
+
+1. Share the AMI's with with Amazon by selecting the AMI's that you've just
+  produced and click the yellow share button.
+
+1. Wait for the scanning status to be succcessful.
+
+1. Update the AMI ID's in the product load form(s) for the marketplace you're
+  updating. You can find the load forms in the Engineering > Marketplace
+  folder on drive.
+
+1. Upload the new load forms in the [AWS Marketplace portal](https://aws.amazon.com/marketplace/management/product-load/).
+
+1. Email <aws-marketplace-seller-ops@amazon.com> to notify them of the changes
+  and that that you'd like a new release done.
+
```

Comment:
Can we add a note here on what one can expect time-wise for a turnaround once the email it set? Will help set expectations with the next person that has to do this.